### PR TITLE
PS-3945: Convert user statistics time columns to floating point

### PR DESCRIPTION
**Pull request by inikep**

### CODE REVIEW

File: sql/sql_show.cc [hunk 1]
```diff
@@ -5502,11 +5500,11 @@ static ST_FIELD_INFO user_stats_fields_info[] = {
      MY_I_S_UNSIGNED, "Total_connections", SKIP_OPEN_TABLE},
     {"CONCURRENT_CONNECTIONS", MY_INT64_NUM_DECIMAL_DIGITS, MYSQL_TYPE_LONGLONG,
      0, MY_I_S_UNSIGNED, "Concurrent_connections", SKIP_OPEN_TABLE},
-    {"CONNECTED_TIME", MY_INT64_NUM_DECIMAL_DIGITS, MYSQL_TYPE_LONGLONG, 0,
+    {"CONNECTED_TIME", MY_INT64_NUM_DECIMAL_DIGITS, MYSQL_TYPE_DOUBLE, 0,
```
Comment (laurynas-biveinis) [inline sql/sql_show.cc:5503]: There are testcases that check I_S schema - please grep the testsuite for the changed columns

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Comment by inikep** [comment]
Re-recorded `mysql-test/suite/funcs_1/r/is_columns_is.result`

File: sql/sql_show.cc (hunk 1 above)
Comment (laurynas-biveinis) [reply sql/sql_show.cc:5503]: MY_INT64_NUM_DECIMAL_DIGITS might work as display length for double too but even then it needs a better mnemonic

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: sql/sql_show.cc (hunk 1 above)
Comment (inikep) [reply sql/sql_show.cc:5503]: I kept `MY_INT64_NUM_DECIMAL_DIGITS` because MariaDB did so.

**Pull request merged**